scaffolding-ruby/lib/scaffolding.sh

```diff
@@ -165,6 +165,7 @@ EOT
 
   env_sh="$pkg_prefix/config/app_env.sh"
   mkdir -p "$(dirname "$env_sh")"
+  echo "export SSL_CERT_FILE='$(pkg_path_for core/cacerts)/ssl/cert.pem'" >> "$env_sh"
```

Comment:
When using `pkg_path_for` in a plan you don't need the origin, so `pkg_path_for cacerts` would suffice. This is a helpful thing to do, since if you change deps from `core/cacerts` to `myorigin/cacerts` when trying things out you don't have to change it in places like this.

It doesn't really matter in this case, but just a PSA.